### PR TITLE
[PWX-5708] Simplifying the docs for pvc-level encryption

### PR DESCRIPTION
**Pull request by piyush-nimbalkar**

### CODE REVIEW

File: scheduler/kubernetes/pvc-encryption.md [hunk 1]
```diff
@@ -65,9 +66,16 @@ spec:
     requests:
       storage: 2Gi
 ```
-The encrypted PVC will use the key `mysql-pvc` under the Kubernetes secret `vol-secrets` in `example` namespace. If the secret key is not present or Portworx does not have permissions to read the secret, then the volume creation will fail until the key is created or the permission granted.
 
-To grant Portworx permisssions to read the `vol-secrets` secret under `example` namespace, do the following:
+The encrypted PVC will use the key `mysql-pvc` under the Kubernetes secret `volume-secrets` in `portworx` namespace. If the secret key is not present, then the volume creation will fail until the key is created.
+
+Out of all the annotations in the above PVC, only `px/secret-name` is mandatory.
```
Comment (disrani-px) [inline scheduler/kubernetes/pvc-encryption.md]: Maybe reword: From the annotations...

Comment (piyush-nimbalkar) [reply scheduler/kubernetes/pvc-encryption.md]: Done
